### ISSUE TITLE
For some reason, my 2.5k contributions got reduced to 800 overnight as per the stats

### ISSUE THREAD

**Issue by ElectroPerf** [anuraghazra/github-readme-stats#1447]
**Bug**
It shows that my contributions reduced drastically overnight.

**Expected behavior**
Should have  shown the same 2.5k contribution

**Screenshots / Live demo link (paste the github-readme-stats link as markdown image)**
Before:+

![IMG_20211108_110025_907](https://user-images.githubusercontent.com/76483474/140689401-e63cfa42-c0ff-413d-9664-60a2636d3eaa.jpg)

After:+

![Screenshot_2021-11-08-10-59-27-469](https://user-images.githubusercontent.com/76483474/140689456-ceb1bb4f-4054-49e3-bd32-1e3af30e49fd.jpeg)

**Additional context**
Ps:- i have my private contribution turned on in the readme.md

<!--

**Comment by xdvrx1** [comment]
oh yeah, I just posted another issue, which `Contributed To` was affected, the count was decreased also. 

And my commits were also decreased, same as yours.

**Comment by xdvrx1** [comment]
I'm observing this ... 

just an update now, it's 2.4k commits, it's decreasing too ...

**Comment by ElectroPerf** [comment]
> I'm observing this ...
> 
> just an update now, it's 2.4k commits, it's decreasing too ...

Yea same

**Comment by rickstaa** [comment]
Related to #1053

**Comment by rickstaa** [comment]
Duplicate of #1515.
